files/entry.js

```diff
@@ -17,99 +18,101 @@ const server = new Server(manifest);
 const initialized = server.init({ env: process.env });
 
 /**
- * @typedef {import('@azure/functions').AzureFunction} AzureFunction
- * @typedef {import('@azure/functions').Context} Context
+ * @typedef {import('@azure/functions').InvocationContext} InvocationContext
  * @typedef {import('@azure/functions').HttpRequest} HttpRequest
+ * @typedef {import('@azure/functions').HttpResponse} HttpResponse
  */
 
-/**
- * @param {Context} context
- */
-export async function index(context) {
-	const request = toRequest(context);
-
-	if (debug) {
-		context.log(
-			'Starting request',
-			context?.req?.method,
-			context?.req?.headers?.['x-ms-original-url']
-		);
-		context.log(`Original request: ${JSON.stringify(context)}`);
-		context.log(`Request: ${JSON.stringify(request)}`);
-	}
-
-	const ipAddress = getClientIPFromHeaders(request.headers);
-	const clientPrincipal = getClientPrincipalFromHeaders(request.headers);
-
-	await initialized;
-	const rendered = await server.respond(request, {
-		getClientAddress() {
-			return ipAddress;
-		},
-		platform: {
-			clientPrincipal,
-			context
+app.http('sk_render', {
+	methods: ['HEAD', 'GET', 'POST', 'DELETE', 'PUT', 'OPTIONS'],
+	/**
+	 *
+	 * @param {HttpRequest} httpRequest
+	 * @param {InvocationContext} context
+	 */
+	handler: async (httpRequest, context) => {
+		if (debug) {
+			context.log(
+				'Starting request',
+				httpRequest.method,
+				httpRequest.headers.get('x-ms-original-url')
+			);
+			context.log(`Request: ${JSON.stringify(httpRequest)}`);
 		}
-	});
 
-	const response = await toResponse(rendered);
+		const request = toRequest(httpRequest);
+
+		const ipAddress = getClientIPFromHeaders(request.headers);
+		const clientPrincipal = getClientPrincipalFromHeaders(request.headers);
+
+		await initialized;
+		const rendered = await server.respond(request, {
+			getClientAddress() {
+				return ipAddress;
+			},
+			platform: {
+				user: httpRequest.user,
```

Comment:
This needs a README update: https://github.com/derkoe/svelte-adapter-azure-swa/blob/7a855e8885385ba12a8a71f603dc6f5e8537b732/README.md#L227-L228